learning/tour-of-beam/terraform-v2/modules/cloud_functions/main.tf

```diff
@@ -0,0 +1,34 @@
+resource "google_cloudfunctions_function" "cloud_function" {
+  count                 = length(var.entry_point_names)
+  name                  = "${var.entry_point_names[count.index]}"
+  runtime               = "go116"
+  project               = var.project_id
+  service_account_email = var.service_account_id
```

Comment:
Is this the only reason we created a SA?

Reply:
Sorry, what reason?
In this context, we need this SA cloud functions will run with it